packages/cli/tests/e2e/frontend/TestCreateTab.tests.ts

```diff
@@ -34,7 +35,7 @@ describe("Create single tab", function () {
     // clean up
     await cleanUp(appName, projectPath, true, false, false);
   });
-  describe("feature flags for API v3", async function () {
+  describe("feature flags for API v3", { testPlanCaseId: 10306848 }, async function () {
```

Comment:
10306848 Local Preview: Check Bot running successfully
Wrong id?

Reply:
sorry. should be 10298738